core/camel-core/src/test/java/org/apache/camel/component/file/FileBatchConsumerMemoryLeakManualTest.java

```diff
@@ -64,7 +69,7 @@ public void process(Exchange exchange) throws Exception {
         });
         context.start();
 
-        Thread.sleep(30 * 1000L);
+        latch.await(30, TimeUnit.SECONDS);
```

Comment:
This is incorrect. There needs to be something to count down the latch.

Reply:
Thanks! 
I have reverted the changes.